src/main/java/com/example/dto/CreateBookRequestDto.java

```diff
@@ -9,9 +13,15 @@
 @Setter
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotNull
```

Comment:
What aboud other String ?)) 
```suggestion
    @NotBlank
```

Reply:
I've though about this. And decided that it may happens situation when book don't have the author. In this case @NotNull better than @NotBlank